includes/wikia/services/sass/Filter/CdnRewriteFilter.class.php

```diff
@@ -27,7 +27,7 @@ public function process( $contents ) {
 
 			// TODO: refactor?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[project-id]: wikia:app